src/@batch-flask/ui/editor/monaco-languages/json/pool.template.json

```diff
@@ -44,11 +44,11 @@
           "title": "The desired number of dedicated compute nodes in the pool.",
           "description": "The desired number of dedicated compute nodes in the pool."
         },
-        "targetLowPriorityNodes": {
+        "targetSpotLowPriorityNodes": {
```

Comment:
The property name is still `targetLowPriorityNodes` to avoid making backwards incompatible changes, just the label has changed.

Reply:
Fixed